ntoskrnl/inbv/inbv.c

```diff
@@ -1095,6 +1121,7 @@ DisplayBootBitmap(IN BOOLEAN TextMode)
         if (TempRotBarSelection != RB_UNSPECIFIED)
         {
             // FIXME: This was added to allow animation start before processor hang
+            LARGE_INTEGER Delay;
```

Comment:
I "localized" in one place all the stuff needed for the "hack".

Reply:
Hmm... I tried to do this before, but it gave me errors about mixed C99 ISO declarations (something like that).

Reply:
It shouldn't, when you put the new variable(s) declaration(s) at the very beginning inside a block.

Reply:
Yep, got it, now it compiles fine :)